tests/unit/dataactbroker/test_fileRoutes.py

```diff
@@ -60,3 +61,98 @@ def test_list_submissions(file_app, database, user_constants, job_constants):
     g.user = user2
     response = file_app.get("/v1/list_submissions/?certified=mixed")
     assert sub_ids(response) == {submissions[2].submission_id}
+
+
+def test_current_page(file_app, database, user_constants, job_constants, monkeypatch):
+    """Test the route to check what the current progress of the submission is at
+    the correct page
+    """
+
+    cgacs = CGACFactory()
+    print(cgacs)
```

Comment:
Extra print statement (also, just name it `cgac` instead of `cgacs` because it's just the one. (make sure this works if it isn't an array, if it needs to be one you can just wrap it in `[]` I think)